cast/src/main.rs

```diff
@@ -31,6 +31,10 @@ struct Cli {
     )]
     accounts_file_path: Utf8PathBuf,
 
+    /// If passed, values will be displayed as integers, otherwise as hexes
+    #[clap(short, long)]
+    int_format: bool,
```

Comment:
it's fine I guess, but maybe `--format int | hex` would be a bit better? the default would be hex